### DISCUSSION TITLE
How do I make a new draft Post in 0.46.x?

### DISCUSSION THREAD

**Discussion by nabeards** [GetPublii/Publii#1731] [Help] [answered]
How do I make a new draft Post in 0.46.x? Before 0.46, I could set the Post type to Draft. Now, it just says to set the published date. I have two Draft Posts from prior to upgrading, but now I can't make a new one. What am I missing to make this work?

**Comment by nabeards** [comment, marked as answer]
Bah, ask it and you find the answer. When you go to Save, you have to click the drop-down and choose Save as Draft, per the docs:

https://getpublii.com/docs/adding-editing-and-deleting-posts.html#savingyourpost

**Reply by candidexmedia** [reply]
That's the way it goes sometimes 😂

Glad you found your answered and shared it with everyone.
